config.json

```diff
@@ -725,6 +725,18 @@
       ]
     },
     {
+      "uuid": "666fda78-851e-4bb0-a416-953528f46bd7",
+      "slug": "spiral-matrix",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 2,
+      "topics": [
+        "algorithms",
+        "control-flow (loops)",
```

Comment:
`control-flow (loops)` should just be `loops` - see [TOPICS.txt](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) for a full list of valid topics.